### PR TITLE
THREESCALE-8778[UI Strings] Description in Application plan introduction on the application plan list is incorrect

### PR DESCRIPTION
**Pull request by lvillen**
**What this PR does / why we need it**:
**Which issue(s) this PR fixes** 

[THREESCALE-8778[UI Strings] Description in Application plan introduction on the application plan list is incorrect](https://issues.redhat.com/browse/THREESCALE-8778)

**Verification steps** 

1. Go to Applications > Applications Plans
2. Confirm the microcopy matches the strings. 

Snapshot: 

Captura de pantalla 2023-01-31 a las 16 30 41

**Special notes for your reviewer**:

### CODE REVIEW

File: app/views/api/application_plans/index.html.slim [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-  | Application plans establish rules like limits, pricing, and features for 
-  | using your API. A developer's application accesses your API 
-  | following the rules of the application plan. Configure application plans 
-  | for specific target audiences with different sets of rules.
+  | Application Plans establish the rules (limits, pricing, features) for 
+  | using your API; every developer's application accessing your API 
+  | will be accessing it within the constraints of an application plan. 
+  | From a business perspective, Application Plans allow you to target different 
+  | audiences by using multiple plans, that is, basic, pro, premium, 
+  | with different sets of rules.
```
Comment (dfennessy) [inline app/views/api/application_plans/index.html.slim:8]: LGTM!

**Review by dfennessy — APPROVED** [review]

**Review by thalesmiguel — DISMISSED** [review]

File: app/views/api/application_plans/index.html.slim [hunk 2]
```diff
@@ -1,9 +1,11 @@
-h1 Application plans
+h1 Application Plans
```
Comment (josemigallas) [inline app/views/api/application_plans/index.html.slim:1]: Wait a second, this looks contradictory. My understanding is that only first letter is capitalized. That's what we have been doing, like with Mapping rules.

**Review by josemigallas — CHANGES_REQUESTED** [review]

File: app/views/api/application_plans/index.html.slim (hunk 2 above)
Comment (dfennessy) [reply app/views/api/application_plans/index.html.slim:1]: Application Plans is correct usage in this context. https://www.patternfly.org/v4/ux-writing/capitalization/#capitalization-across-red-hat-uis

**Review by josemigallas — APPROVED** [review]

**Review by josemigallas — CHANGES_REQUESTED** [review]
🙅🏻‍♂️

**Comment by dfennessy** [comment]
> 🙅🏻‍♂️

More speed less haste. It should be good now  :grimacing:

**Review by dfennessy — CHANGES_REQUESTED** [review]

**Review by josemigallas — APPROVED** [review]

**Review by dfennessy — APPROVED** [review]
LGTM

**Pull request merged**